packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -46,10 +49,54 @@ export default class Meeting extends StatelessSparkPlugin {
     this.sipUri = null;
   }
 
+  /**
+   * invite a guest to the call that isn't normally part of this call
+   * @param {Object} invitee with {emailAddress or email}
+   * @param {Boolean} alertIfActive optional
+   * @returns {Promise}
+   * @memberof Meeting
+   */
+  invite(invitee, alertIfActive = true) {
+    return this.participants.addParticipant(invitee, alertIfActive);
+  }
+
+  /**
+   * admit the guest(s) to the call once they are waiting
+   * @param {Array} participantIds
+   * @returns {Promise}
+   * @memberof Meeting
+   */
+  admit(participantIds) {
+    return this.participants.admitParticipants(participantIds);
+  }
+
+  parseLocus(locus) {
+    // parse all the object and attach it
+    this.locusUrl = locus.url;
+    this.type = this.getLocusType(locus.info.locusTags);
+    this.partner = locus.participants.find((p) => (locus.self && p.identity !== locus.self.identity)
+      && (locus.participants.length <= 2 || (p.type === 'USER' && !p.removed))) || this.partner;
```

Comment:
Is this `USER` a part of the constant file? If it isn't, no need to add. But if it is and is used elsewhere, it may be good to use here too.